macos/Ghostty.xcodeproj/project.pbxproj

```diff
@@ -405,7 +411,7 @@
 				A5985CE52C33060F00C57AD3 /* man */,
 				A5A1F8842A489D6800D1E8BC /* terminfo */,
 				FC5218F92D10FFC7004C93E0 /* zsh */,
-				9351BE8E2D22937F003B3499 /* nvim */,
+				9351BE8E2D22937F003B3499 /* vim */,
```

Comment:
Hi! I noticed that in this PR, both vim and nvim folders are renamed to "vim" in the Xcode project references. While the final build output still correctly maintains the distinction between vim and nvim folders, I'm curious about the motivation behind this change in the project structure.

I see that both folders contain identical content at the moment, but having two identically named "vim" folders in the Resources group (as shown in the Xcode project navigator) might be a bit confusing for future maintenance. Could you share the reasoning behind this unification of names in the Xcode project?

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/59c94f09-6031-4b85-a5ac-d0e3b7a256f4" />

Reply:
Hm. I didn’t do this on purpose at all. So this was somehow a mistake. I’ll take a look!